cpp/ps0000/PS000.cpp

```diff
@@ -1,9 +1,10 @@
-#include <iostream>
+#include <iostream> // 2/2
 
 using namespace std;
 
-int main() // () not (void)
+int main() // 1/1
 {
-    cout << "Hello world!\n"; // don�t forget ;
-    return 0; // return 0 not 1
+    cout << "Hello, world!\n"; // don’t forget ; // 3/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [ ] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line